musicbrainz-server/recipes/production.rb

```diff
@@ -15,7 +15,17 @@
 daemontools_service "musicbrainz-server" do
   directory "/home/musicbrainz/svc-musicbrainz-server"
   template "musicbrainz-server"
-  variables :nproc => node['musicbrainz-server']['nproc']
+  variables :nproc => node['musicbrainz-server']['nproc-website']
+  action [:enable,:start]
+  subscribes :restart, "git[/home/musicbrainz/musicbrainz-server]"
+  subscribes :restart, "template[/home/musicbrainz/musicbrainz-server/lib/DBDefs.pm]"
+  log true
+end
+
+daemontools_service "musicbrainz-ws" do
+  directory "/home/musicbrainz/svc-musicbrainz-ws"
+  template "musicbrainz-server"
```

Comment:
Hm, a separate one probably makes sense. I don't understand the mechanism by which it finds the `sv-*.erb` files in musicbrainz-server/templates/default/, would changing this to "musicbrainz-ws" do that automatically?